pkg/apis/clusteroperator/v1alpha1/types.go

```diff
@@ -81,6 +81,14 @@ type AWSClusterSpec struct {
 	// credentials
 	AccountSecret corev1.LocalObjectReference `json:"accountSecret"`
 
+	// SSHSecret refers to a secret that contains the ssh private key to access
+	// EC2 instances for this cluster
+	SSHSecret corev1.LocalObjectReference `json:"sshSecret"`
+
+	// SSHSecretName is the name of the secret to use for SSH access to EC2
+	// instances in this cluster
+	SSHSecretName string `json:"sshSecretName"`
```

Comment:
Why both?

Reply:
We need both -- this is the name of the ssh secret object in AWS (ie. "libra"). The reference is for a secret that contains the private key for that ssh secret.

Reply:
These names are super confusing.  We use SecretName to mean "name of a Kube secret".  Might be better to use something instead of `SecretName` and improve the godoc.